hazelcast/src/main/java/com/hazelcast/quorum/QuorumListener.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.quorum;
+
+import java.util.EventListener;
+
+/**
+ * Listener to get notified when a quorum state is changed
+ */
+public interface QuorumListener extends EventListener {
+
+    void onQuorumSuccess(QuorumEvent quorumEvent);
```

Comment:
Why not a single method 'handle'? In the handle method you can see what type of event it is.